assets/ts-examples/google.ts

```diff
@@ -7,7 +7,7 @@ const home: NightwatchTests = {
 
   'Google search test': () => {
     browser
-      .setValue('input[name=q]', 'nightwatchjs')
```

Comment:
I have a suggestion. What if we update all our examples to use newer APIs? This way, new users can see working examples and won't be left in the dark.

Reply:
We can update the apis in a later release. 
Tracking here: https://github.com/nightwatchjs/nightwatch-setup-tools/issues/2